commit e2e2f5a7b783ee8a27d861369ba9de984b7d7221 (refs/changes/03/5503/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-02-06T19:46:54+00:00 (8 months ago)
    
    TOOLS-2174 agent-cache 'clone' should use '--' when switching branches
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/deps/eng b/deps/eng
index 4443fcd..7c472bc 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 4443fcd38fe1682556e1851df97c7d652d078f3d
+Subproject commit 7c472bc495ba44ac4cdde0af50ff627021942524
